src/libstore/ssh-store.hh

```diff
@@ -0,0 +1,28 @@
+#pragma once
+///@file
+
+#include "common-ssh-store-config.hh"
+#include "store-api.hh"
+#include "remote-store.hh"
+
+namespace nix {
+
+struct SSHStoreConfig : virtual RemoteStoreConfig, virtual CommonSSHStoreConfig
```

Comment:
Any comments here you can add?
I'm not familiar with what makes this experimental vs. legacy?

Should the legacy have \deprecated or @deprecated (whatever is the pattern in doxygen)

Reply:
I just moved this code. the legacy ssh uses a different protocol, but this one uses the same protocol as the Nix daemon. It probably shouldn't say "experimental" as its not an experimental feature (it predates those), but I think that's best done in a separate PR.